docs/articles/faq/README.md

```diff
@@ -0,0 +1,253 @@
+
+* it has a completely new API and offers a new approach to writing tests;
+* it has no UI to manage and run tests. You can run test from the CLI or from a node.js module;
+* it's more convenient to integrate into node.js development workflow
+* it has some new features (like es6 support, flexible selectors and smart assertions, authentication features etc.)
+
+You can learn more about the open-source version at [https://devexpress.github.io/testcafe](https://devexpress.github.io/testcafe).
+
+However, we didn't abandon our aspirations to create a competitive proprietary product.
+We plan to release a completely new commercial version based on the revised open-source TestCafe.
+Stay tuned to [https://testcafe.devexpress.com](https://testcafe.devexpress.com) to keep in touch
+about the new product that will be called ***TestCafe Studio***.
+
+### Which browsers does TestCafe support? What are the exact supported versions?
+
+We test TestCafe against a limited number of browsers. We call these browsers *officially* supported.
+You can find the list of these browsers in [our documentation](../documentation/using-testcafe/common-concepts/browser-support.md).
+
+At the same time, TestCafe should work with almost every modern browser with extensive support for HTML5.
+You can check if TestCafe works in the browser of your interest by yourself.
+
+As to the exact browser versions, we test TestCafe against two newest versions
+of each officially supported browser (unless the exact version is specified explicitly
+in the supported browser list). However, TestCafe actually works with many older versions
+since we usually refrain from using cutting edge features in TestCafe code. In fact,
+TestCafe should work in any officially supported browser released in the last three years.
+
+### Can I use third-party modules in tests?
+
+Yes, you can import them to test files just like you would do with a regular node.js module.
+
+On the server side, you only need to use the `import` statement.
+
+```js
+import fs from 'fs';
+
+fixture `fixture`
+   .page('http://localhost/testcafe/');
+
+test('test', async t => {
+   // var filePath = ...;
+   await t.expect(fs.existsSync(filePath)).ok();
+});
+```
+
+On the client side, use `t.eval` to include the desired module in the test.
+Then you can use this module inside client functions and selectors.
+
+```sh
+test('test', async t => {
+    // eval jquery code to add jQuery to the page
+    await t.eval(new Function(fs.readFileSync('./jquery.js').toString()));
+
+    var clientFunction = ClientFunction(() => {
+        // You can use $ here
+        return $('div').text();
+    });
+
+    var text = await clientFunction();
+});
+```
+
+### How do I work with configuration files and environment variables?
+
+As TestCafe works with zero configuration, it does not have any config files
+where you can place your custom variables. However, you can always introduce
+your own configuration file and import it to test code.
```

Comment:
TestCafe does not have any config files where you can place custom variables. But you can introduce your own configuration file and import it to test code.